content/posts/2019/2019-07-05-littlest-cuda-hub.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Setting up GPU Data Science environments for hackathons"
+subtitle: "AKA Littlest Cuda Hub"
+date: 2019-07-05T00:00:00+00:00
+draft: true
+author: "Jacob Tomlinson"
+categories:
+  - blog
+tags:
+  - rapids
+  - iris
+  - aws
+  - jupyterhub
+  - cuda
+  - gpu
+thumbnail: jupyter
+---
+
+## Background
+
+In my first week working at Nvidia I have been spending some time with my previous colleagues at the Met Office to explore how the two organisations can collaborate. 
+
+The theme for the week was to run a hackathon to explore how GPUs could be used to accelerate existing tools and workflows within the Met Office. To begin trying things out we needed a collaborative environment with access to GPUs. For this the Met Office kindly provided a GPU EC2 instance on AWS and we set up RAPIDS and the [Littlest JupyterHub](https://tljh.jupyter.org/en/latest/) (tljh) to expose it to collaborators.
```

Comment:
You might say something here about the target audience

"We had 10-20 people who knew Python well, but who didn't know much about GPUs other than that they were curious."

This might help readers identify themselves with the situation.